android/java/org/chromium/chrome/browser/privacy/settings/BravePrivacySettings.java

```diff
@@ -29,68 +34,139 @@
+            PREF_DO_NOT_TRACK, PREF_CLOSE_TABS_ON_EXIT, PREF_SEND_P3A, PREF_SEARCH_SUGGESTIONS,
+            PREF_AUTOCOMPLETE_TOP_SITES, PREF_AUTOCOMPLETE_BRAVE_SUGGESTED_SITES,
+            PREF_USAGE_STATS, PREF_PRIVACY_SANDBOX
+
+    };
 
     private final PrefService mPrefServiceBridge = UserPrefs.get(Profile.getLastUsedRegularProfile());
     private final ChromeManagedPreferenceDelegate mManagedPreferenceDelegate =
             createManagedPreferenceDelegate();
     private ChromeSwitchPreference mSearchSuggestions;
+    private ChromeSwitchPreference mCanMakePayment;
+    private BraveDialogPreference mAdsTrakersBlockPref;
+    private BraveDialogPreference mBlockCrosssiteCookies;
     private ChromeSwitchPreference mAutocompleteTopSites;
     private ChromeSwitchPreference mAutocompleteBraveSuggestedSites;
-    private ChromeBaseCheckBoxPreference mHttpsePref;
+    private ChromeSwitchPreference mHttpsePref;
     private ChromeBaseCheckBoxPreference mIpfsGatewayPref;
     private ChromeBaseCheckBoxPreference mAdBlockPref;
-    private ChromeBaseCheckBoxPreference mFingerprintingProtectionPref;
-    private ChromeBaseCheckBoxPreference mCloseTabsOnExitPref;
-    private ChromeBaseCheckBoxPreference mSendP3A;
+    private BraveDialogPreference mFingerprintingProtectionPref;
+    private ChromeSwitchPreference mBlockScriptsPref;
+    private ChromeSwitchPreference mCloseTabsOnExitPref;
+    private ChromeSwitchPreference mSendP3A;
     private PreferenceCategory mSocialBlockingCategory;
     private ChromeSwitchPreference mSocialBlockingGoogle;
+    private ChromeSwitchPreference mHttpsEverywhere;
     private ChromeSwitchPreference mSocialBlockingFacebook;
     private ChromeSwitchPreference mSocialBlockingTwitter;
     private ChromeSwitchPreference mSocialBlockingLinkedin;
     private ChromeBasePreference mWebrtcPolicy;
+    private ChromeSwitchPreference mClearBrowsingDataOnExit;
 
     @Override
     public void onCreatePreferences(Bundle savedInstanceState, String rootKey) {
         super.onCreatePreferences(savedInstanceState, rootKey);
 
+        // override title
+        getActivity().setTitle(R.string.brave_shields_and_privacy);
+
         SettingsUtils.addPreferencesFromResource(this, R.xml.brave_privacy_preferences);
 
-        mHttpsePref = (ChromeBaseCheckBoxPreference) findPreference(PREF_HTTPSE);
-        mHttpsePref.setOnPreferenceChangeListener(this);
+        mHttpsePref = (ChromeSwitchPreference) findPreference(PREF_HTTPSE);
+        mHttpsePref.setOnPreferenceChangeListener(this);        
+
+        mCanMakePayment = (ChromeSwitchPreference) findPreference(PREF_CAN_MAKE_PAYMENT);
+        mCanMakePayment.setOnPreferenceChangeListener(this);
+
+        // mIpfsGatewayPref = (ChromeBaseCheckBoxPreference) findPreference(PREF_IPFS_GATEWAY);
+        // mIpfsGatewayPref.setOnPreferenceChangeListener(this);
```

Comment:
we don't need to listen for that check anymore?

Reply:
They have been removed from the settings. I left them for future reference, but if it's not useful i should delete them